frontend/src/css/App.css

```diff
@@ -88,9 +88,9 @@ p {
   text-align: center;
 }
 
-.btn-backtotop {
-  border: 1px solid #009fae;
-  color: #009fae;
+.btn-backtotop:hover {
+  color: white;
```

Comment:
I don't think you need to specify the text color here, since the Aurora class already put white.
I tested it and it looks exactly the same:

**As it is right now:**

![image](https://user-images.githubusercontent.com/23021242/53564887-08909580-3b26-11e9-946e-b38e3c76b918.png)

**As it is right now [hover]:**

![image](https://user-images.githubusercontent.com/23021242/53564817-d848f700-3b25-11e9-8cab-a156490f4b8b.png)

**Without specifying the color:**

![image](https://user-images.githubusercontent.com/23021242/53564920-1b0acf00-3b26-11e9-8273-24489dc3c829.png)


**Without specifying the color [hover]:**

![image](https://user-images.githubusercontent.com/23021242/53564781-c6675400-3b25-11e9-8fa4-7239460ff64f.png)